Softeq.XToolkit.Common.Droid.Tests/TextFilters/ForbiddenCharsInputFilterTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Android.Text;
+using Java.Lang;
+using Softeq.XToolkit.Common.Droid.TextFilters;
+using Xunit;
+
+namespace Softeq.XToolkit.Common.iOS.Tests.TextFilters.ForbiddenCharsFilterTests
+{
+    public class ForbiddenCharsInputFilterTests
+    {
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData("a")]
+        [InlineData("aaa")]
+        [InlineData("aA%@2-")]
+        public void Ctor_WhenCalledWithAnyChars_CreatesFilter(string chars)
```

Comment:
Case with `chars = null` can be moved to a separate test method to avoid null-check in every test.

Reply:
Done